src/Airy/cairy.jl

```diff
@@ -56,29 +56,47 @@ function _airyai(z::Complex{T}) where T <: Union{Float32, Float64}
             return 1 / z
         end
     end
+
+    x, y = reim(z)
+
+    check_conj = false
+    if y < zero(T)
+        z = conj(z)
+        check_conj = true
+    end
+
+    if airy_large_argument_cutoff(x, y)
+        r = airyaix_large_args(z)[1] * exp(-2 * z * sqrt(z) / 3)
+    elseif airyai_levin_cutoff(x, y)
+        r = airyaix_levin(z, Val(17)) * exp(-2 * z * sqrt(z) / 3)
+    elseif airyai_power_series_cutoff(x, y)
+        r = airyai_power_series(z)[1]
+    else
+        r = cispi(one(T)/3) * _airyai(-z*cispi(one(T)/3))  + cispi(-one(T)/3) * _airyai(-z*cispi(-one(T)/3))
```

Comment:
you should check that the compiler is smart enough here (and if not manually tell it that `cispi(1/3) = conj(cispi(-1/3))`